.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
**TODO** Got **lots** of red for linter errors when I pulled down your code. The reason is because this file needs to be explicitly named as `.eslintrc.json`.  Fix this and you'll get your 1 point back for linter config.